script/deployers/MumbaiMangroveTestnetDeployer.s.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.13;
+
+import {ToyENS} from "mgv_lib/ToyENS.sol";
+
+import {Deployer} from "mgv_script/lib/Deployer.sol";
+
+import {MumbaiMangroveDeployer} from "mgv_script/core/deployers/MumbaiMangroveDeployer.s.sol";
+import {MumbaiMangroveOrderDeployer} from
+  "mgv_script/strategies/mangroveOrder/deployers/MumbaiMangroveOrderDeployer.s.sol";
+import {
+  MumbaiKandelSeederDeployer,
+  KandelSeeder,
+  AaveKandelSeeder
+} from "mgv_script/strategies/kandel/deployers/MumbaiKandelSeederDeployer.s.sol";
+
+import {ActivateMarket, IERC20} from "mgv_script/core/ActivateMarket.s.sol";
+import {ActivateMangroveOrder, MangroveOrder} from "mgv_script/strategies/mangroveOrder/ActivateMangroveOrder.s.sol";
+import {KandelSower, IMangrove} from "mgv_script/strategies/kandel/KandelSower.s.sol";
+
+import {Mangrove} from "mgv_src/Mangrove.sol";
+import {MgvReader} from "mgv_src/periphery/MgvReader.sol";
+
+/**
+ * Deploy and configure a complete Mangrove testnet deployment:
+ * - Mangrove and periphery contracts
+ * - MangroveOrder
+ * - KandelSeeder and AaveKandelSeeder
+ * - open markets: DAI/USDC, WETH/DAI, WETH/USDC
+ */
+contract MumbaiMangroveTestnetDeployer is Deployer {
+  function run() public {
+    runWithChainSpecificParams();
+    outputDeployment();
+  }
+
+  function runWithChainSpecificParams() public {
+    // Deploy Mangrove
+    new MumbaiMangroveDeployer().runWithChainSpecificParams();
```

Comment:
It seems somewhat arbitrary to have some deployments in separate scripts, while kandel-sowing and market activations below are not in their own scripts.
The scripts reduce cognitive overhead, but are we expecting them to be reused?